Formula/rex.rb

```diff
@@ -217,22 +232,23 @@ def install
 
     perl_build
     (libexec/"lib").install "blib/lib/Rex", "blib/lib/Rex.pm"
-
-    # Use system Perl
-    inreplace("bin/rexify") { |s| s.gsub!(/^#!perl$/, "#!/usr/bin/env perl") }
+    inreplace "bin/rex", "#!/usr/bin/env perl", "#!/usr/bin/perl"
+    inreplace "bin/rexify", "#!perl", "#!/usr/bin/perl"
 
     %w[rex rexify].each do |cmd|
       libexec.install "bin/#{cmd}"
-      chmod 0755, libexec/cmd
+      chmod 0555, libexec/cmd
       (bin/cmd).write_env_script(libexec/cmd, :PERL5LIB => ENV["PERL5LIB"])
       man1.install "blib/man1/#{cmd}.1"
     end
   end
 
   test do
-    assert_match "\(R\)\?ex #{version}", shell_output("#{bin}/rex -v"), "rex -v is expected to print out Rex version"
+    assert_match "\(R\)\?ex #{version}", shell_output("#{bin}/rex -v"),
```

Comment:
I can pull out the formatting issue from the revision bump commit if you like, but at some point it kind of seems like you're creating unnecessary style deviations for the sake of unnecessary style deviations. It's your repo to run, but Homebrew literally emphases style fixes as a potential contribution point.